### PR TITLE
[ Bugfix & Docs ] Overflow issue on Explore events closes  #1561 & Adding documentation for explore_events.dart

### LINKED ISSUE #1561: Overflow Issue on Events Page
**Describe the bug**
Overflow in the top bar of Events Screen

**To Reproduce**

**Screenshots**
photo_2023-02-24_15-28-27

### PR DESCRIPTION
**Pull request by SiddheshKukade**
**What kind of change does this PR introduce?**

Bugfix

**Issue Number:**

Fixes #1561

**Did you add tests for your changes?**
Not required
**Snapshots/Videos:**

**If relevant, did you update the documentation?**

Not required

**Summary**
Reduced the size of the row to remove the overflow
**Does this PR introduce a breaking change?**
No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by literalEval** [comment]
@SiddheshKukade from today on, contributors are required to write documentation for each file they are modifying in their PR. Please look at my pinned message in #talawa Slack channel for more information.

**Comment by literalEval** [comment]
@SiddheshKukade do you get lint errors in your IDE now ? Do you think we need a detailed guide on the new lint rules and the format we are expecting ? Or is it self explanatory from the warning generated ?

**Comment by palisadoes** [comment]
Please add it here. It's the new process and is documented

File: lib/views/after_auth_screens/events/explore_events.dart [hunk 1]
```diff
@@ -235,7 +234,14 @@ class ExploreEvents extends StatelessWidget {
...
 
-  // dropDownList returns a widget of a drop down list of events type.
+  /// Shows a list of dropdown taken from  `model` and `context`.
+  ///
+  /// params:
+  /// * [model] contains the events data
+  /// * [context] the overall context of UI
+  ///
+  /// returns:
+  /// * [Widget]: the dropdown
```
Comment (SiddheshKukade) [inline lib/views/after_auth_screens/events/explore_events.dart:244]: Hey @literalEval I followed the guide on on docs.talawa.io 
but this is still giving the error.
image

Can you find any mistakes here?

Comment (literalEval) [reply lib/views/after_auth_screens/events/explore_events.dart:244]: @SiddheshKukade it's 

```
/// params:
/// * `model`: ....... 
```

and not

```
/// params:
/// * [model]: ....... 
```

basically replace `[` and `]` with `` ` ``

Comment (SiddheshKukade) [reply lib/views/after_auth_screens/events/explore_events.dart]: Now it passes. I actually used that `[ ]` from the dart official docs.

**Review by noman2002 — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**